C/CvxCompress/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "CvxCompress"
+version = v"1.0.0"
+
+# Collection of sources required to build CvxCompress
+sources = [
+    GitSource(
+        "https://github.com/ChevronETC/CvxCompress.git",
+        "55e072862196e917e3ecece81f014e93d60cdf81"
+    )
+]
+
+# Bash recipe for building across platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/CvxCompress
+make -j${nproc}
+mkdir -p ${libdir}
+cp libcvxcompress.so ${libdir}/libcvxcompress.${dlext}
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+  Linux(:x86_64, libc=:glibc)
```

Comment:
The decision of which platforms to build for is up to you, but maybe you want to try to build for all `supported_platforms()`?  Or you only care about this one?

Reply:
This was the only platform I was sure it would build on, but I am game to try the full set and pair down, I was a little worried about using too much of the available AzureDevOps build resources.

Reply:
Again, you choose the platforms for which to build, but in case you want to try to build for the other x86_64 (the others I think are out of the table) I have an idea of what to try to do :wink:

Reply:
I'm game for pretty much anything :D I don't think the performance can be guaranteed outside of the x86_64 with glibc, but more is always better

Reply:
Ok, I thought it was easier, but not really, I had to patch the makefile to be able to compile natively the utility that you run to generate the files.  Windows seems to require too much work than what I'd like to spend now.  I'm going to merge this as it is now, if you want to change something you can always open a new pull request!

Reply:
Awesome! I appreciate your help!